releases-diff/test/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM prestashop/prestashop:1.7
```

Comment:
Is there a way to do a FROM:build-add-blackfire ??
To avoid duplicating the blackfire setup ?

Reply:
By setting the same FROM statement, and the same blackfire install, the cache is automatically used on the second build.

I made the change in the last commit.